Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Culling.h

```diff
@@ -257,10 +257,10 @@ namespace AZ
             void SetOcclusionPlanes(const OcclusionPlaneVector& occlusionPlanes) { m_occlusionPlanes = occlusionPlanes; }
 
             //! Notifies the CullingScene that culling will begin for this frame.
-            void BeginCulling(const AZStd::vector<ViewPtr>& views);
+            void BeginCulling(const Scene& scene, const AZStd::vector<ViewPtr>& views);
```

Comment:
If you are updating the signature, you can make these parameters a span.
```suggestion
            void BeginCulling(const Scene& scene, AZStd::span<const ViewPtr> views);
```